src/pages/Charcuterie.js

```diff
@@ -1,51 +1,59 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { H1, H2, H3, P, A } from '../components/Typography'
 import { Card, Button, SecondaryButton } from '../components/Common.js'
 import Accordion from '../components/Accordion'
 import Countdown from '../containers/Countdown'
 import Livestream from '../components/Livestream'
 import JudgingCard from '../components/JudgingCard'
+import Checkbox from '../components/Checkbox'
 
-export default () => (
-  <>
-    <H1>Charcuturie</H1>
-    <>
-      <H1>This is an h1.</H1>
-      <H2>This is an h2.</H2>
-      <H3>This is an h3.</H3>
-      <P>
-        <A href="https://en.wikipedia.org/wiki/Charcuterie">Charcuterie</A> most often consists of a
-        variety of meats and cheeses, often paired with crackers, fruit, nuts, and spreads. An ideal
-        charcuterie board has a good balance of flavors and textures and has foods that contrast and
-        complement each other's taste. It's some really long text. I'm really writing this way later
-        than I should be. Is this what it's like to sell your soul to nwPlus?{' '}
-      </P>
-    </>
+export default () => {
+  const [checked, setChecked] = useState(false)
+
+  return (
```

Comment:
Did not delete anything! Just added the state here and added section displaying checkboxes starting line 54.